lib/truffle/truffle/cext_ruby.rb

```diff
@@ -20,7 +20,6 @@ def rb_define_method(mod, name, function, argc)
       raise ArgumentError, "arity out of range: #{argc} for -2..15"
     end
 
-    wrapper = RB_DEFINE_METHOD_WRAPPERS[argc]
```

Comment:
It's used on line 40, isn't it?

Reply:
Looks like it. Sorry. IntelliJ said it was unused and I ran the test suite with the line removed. I'll drop that commit.